pkg/operator/ceph/object/rgw.go

```diff
@@ -317,3 +318,27 @@ func buildDNSEndpoint(domainName string, port int32, secure bool) string {
 	}
 	return fmt.Sprintf("%s://%s:%d", httpPrefix, domainName, port)
 }
+
+// GetTLSCACert fetch cacert for internal RGW requests
+func GetTLSCACert(objContext *Context, objectStoreSpec *cephv1.ObjectStoreSpec) ([]byte, error) {
```

Comment:
Nit. I think this reads a little more easily as `GetTlsCaCert` since the two acronyms (TLS and CA) run together as written.